### DISCUSSION TITLE
Problem to plot Misorientation Distribution Axis (MADs)

### DISCUSSION THREAD

**Discussion by 90ari** [mtex-toolbox/mtex#2051] [Ask Anything] [answered]
Hi,
I am Arindam Sarkar. I am new user of mtex and still learning mtex.

I am using matlab R2018a. I am trying plot MADs for magnetite.  I am running a code (mentioned below). Interestingly, this code is running for all minerals except magnetite and showing error (attached as image). If you kindly help me regarding this problem, I'll be highly obliged.

Code:
```
%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = {...
  'notIndexed',...
  crystalSymmetry('m-3m', [8.400 8.400 8.400], [90.000, 90.000, 90.000]*degree, 'X||a*', 'Y||b', 'Z||c', 'mineral', 'Magnetite', 'color', 'blue'),...
  crystalSymmetry('-3', [5.035 5.035 13.751], [90.000, 90.000, 120.000]*degree, 'X||a*', 'Y||b', 'Z||c', 'mineral', 'Hematite', 'color', 'green')};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outofPlane');

%% Specify File Names

% path to files

pname = 'D:\EBSD\EBSD_Maganetite\CPRCTFFiles_Report\CPRCTFFiles_Sample002\Selected Magnatite File\Project';

% which files to be imported

fname = [pname '\Project 1 Specimen 1 Site 6 Map Data 2.ctf'];

%% Import the Data

% create an EBSD variable containing the data
ebsd = EBSD.load(fname,CS,'interface','ctf',...
  'convertEuler2SpatialReferenceFrame');
% perform grain segmentation
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'),'threshold',10*degree);

% remove small grains
ebsd(grains(grains.grainSize<5)) = [];

% repeat grain reconstruction
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'),'threshold',10*degree);

% smooth the grain boundaries a bit
grains = smooth(grains,5);

%% first compute grains using a separate innerBoudary and outer-boundary angle
% grains
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd,'angle',[10 1]*degree,'boundary','tight');

%% grains of specific phase
phase = 'Magnetite';
g = grains(phase);

%% Discrete low-angle (1–10°) boundary misorientation analysis

% Note, the particular approach used here requires that you specify a
% second angle for inner-boundaries during grain reconstructuion (as above)

% get all the subgrain/inner boundaries from the grainset
subB = g.innerBoundary;

% boundaries of the phase of interest (forsterite in our example case)
sub = subB(phase,phase);

% condition for only misorientations with angles of 2–15° [Arindam here you
% can change the rage of angles accordin to your interest]
condLAB = sub.misorientation.angle>=2*degree & sub.misorientation.angle<15*degree;

% the "low-angle" boundaries of interest
lab = sub(condLAB);

%% plot in crystal reference frame 
figure,
plotAxisDistribution(lab.misorientation,'antipodal','lower','smooth','halfwidth',10*degree,'figSize','small')
mtexColorbar('Title','M.U.D.');
```
![Capture](https://github.com/mtex-toolbox/mtex/assets/157532972/cdceb848-55b8-4f7c-b691-6d260e9e27ab)
 Thanking you,
Regards,
Arindam Sarkar

**Comment by kilir** [comment]
Hi,

it seems you are using the mtex>=6 which is a beta version, can you please try the latest release version and report back?

Also, the second calcGrains is obviously not needed and smoothing boundaries should occur after computing grains.

Cheers,

Rüdiger

**Comment by zmichels** [comment]
I get this issue/error with cubic minerals when plotting using 'fundamentalRegion'.

**Comment by 90ari** [comment]
Respected Sir,

Now, I am using  [MTEX 5.10.2] and facing same problem. 
![Capture](https://github.com/mtex-toolbox/mtex/assets/157532972/ff22c255-f652-465b-8fab-aacab12d9c0a)
Thnking you,
Regards,
Arindam

**Reply by kilir** [reply]
Ok, I can confirm this issue occurs with cubic symmetry.

**Comment by 90ari** [comment]
Respected Sir,
Is there any solution? If you guide me I'll be highly obliged.

Thanking you,
Regards,
Arindam Sarkar

**Reply by zmichels** [reply]
Maybe Rudiger will know a solution. I haven't before seen plotAxisDistribution work with cubic phases, and have received this error for a couple years at least... so I'm guessing it isn't a trivial issue.

you could try a standard plot() command instead of plotAxisDistribution(), and then instead of using the misorientations, you can plot/use axis(misorientations).

**Comment by ralfHielscher** [comment, marked as answer]
do
```matlab
plotAxisDistribution(lab.misorientation,'antipodal','smooth','halfwidth',10*degree,'figSize','small')
```
i.e. without the option "lower". It is conflicting with the default fundamental sector for cubic symmetry.

Ralf.

**Comment by 90ari** [comment]
Respected Ralf Hielscher and Zachary Michels,

Thank you so much for your kind support. I am blessed that you took this issue in a serious account. Now, it is working. Thank you so much again.

Regards,
Arindam Sarkar
